pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2047,20 +1977,50 @@ def predict_model(
 
         """
 
-        return super().predict_model(
-            estimator=estimator,
-            data=data,
-            probability_threshold=None,
-            encoded_labels=True,
-            round=round,
-            verbose=verbose,
-        )
+        # return super().predict_model(
+        #     estimator=estimator,
+        #     data=data,
+        #     probability_threshold=None,
+        #     encoded_labels=True,
+        #     round=round,
+        #     verbose=verbose,
+        # )
+
+        if fh is None:
+            fh = self.fh
+
+        try:
+            return_vals = estimator.predict(
+                X=data, fh=fh, return_pred_int=return_pred_int, alpha=alpha
+            )
+        except NotImplementedError:
+            # Most likely prediction interval has not been implemented.
+            # Hence try by explicitly setting it to False
+            return_vals = estimator.predict(
+                X=data, fh=fh, return_pred_int=False, alpha=alpha
+            )
+        if isinstance(return_vals, tuple):
+            # Prediction Interval is returned
+            #   First Value is a series of predictions
+            #   Second Value is a dataframe of lower and upper bounds
+            result = pd.DataFrame(return_vals[0], columns=["y_pred"])
+            result = result.join(return_vals[1])
```

Comment:
is this join based on index value ?

Reply:
Yes, both the prediction series and interval dataframe have the same indices.